cms/models/pluginmodel.py

```diff
@@ -329,6 +343,13 @@ def get_position_in_placeholder(self):
         """
         return self.position + 1
 
+    def get_warning_on_delete(self):
```

Comment:
Since most of these methods are intended as hooks and all of them start with "get_", I think we should keep this convention. Maybe we should change it to "get_delete_warning_msg".